internal/apidiff/apidiff.go

```diff
@@ -215,6 +214,23 @@ func diff(m manifest, modDir, imp, pkg, base string) (string, error) {
 	return out, err
 }
 
+func checkAllowBreakingChange(commit string) bool {
```

Comment:
Annoying comment: mind adding a unit test?

Reply:
Not annoying at all, done!